src/main/resources/com/puppycrawl/tools/checkstyle/messages_ja.properties

```diff
@@ -0,0 +1,15 @@
+general.fileNotFound=ファイルが見つかりません！
+general.exception=例外が発生しました - {0}
+
+#
+# for DefaultLogger.java
+#
+DefaultLogger.addException=エラー監査 {0}
+DefaultLogger.auditStarted=監査を開始しています...
+DefaultLogger.auditFinished=監査は行われ。
+
+PackageObjectFactory.unableToInstantiateExceptionMessage=''{0}'' クラスをインスタンス化することができません、\
+  それは としてインスタンス化することもできません {1}. \
+  そのクラス名が正規名として指定されている再確認または短い名前の使用 \
+  http://checkstyle.sourceforge.net/config.html#Packages を設定する方法をお読みください。\
```

Comment:
We need to escape `#` here, because otherwise it starts comment. Please also recheck other property files